src/sonic-config-engine/tests/test_cfggen.py

```diff
@@ -894,18 +894,43 @@ def test_minigraph_voq_inband_port(self):
         output_dict = utils.to_dict(output.strip())
         self.assertDictEqual(
             output_dict['Ethernet-IB0'],
-            {'lanes': '133', 
-            'alias': 'Recycle0', 
-            'description': 'Recycle0', 
-            'mtu': '9100', 
-            'tpid': '0x8100', 
-            'pfc_asym': 'off', 
-            'admin_status': 'up',
-            'fec': 'rs',
-            'speed': '100000'
+            {   'admin_status': 'up',
+                'alias': 'Recirc0/1',
+                'coreId': '1',
+                'corePortId': '222',
+                'description': 'Recirc0/1',
+                'index': '52',
+                'lanes': '222',
+                'mtu': '9100',
+                'numVoq': '8',
+                'pfc_asym': 'off',
+                'role': 'Inb',
+                'speed': '400000',
+                'tpid': '0x8100'
             }
```

Comment:
Reformat the python dict, same for below function.

Reply:
fixed in latest commit